tests/hazmat/primitives/test_dh.py

```diff
@@ -18,6 +19,11 @@
 from ...utils import load_nist_vectors, load_vectors_from_file
 
 
+skip_no_dhx = pytest.mark.skipif(
```

Comment:
Thanks for the tip @reaperhulk :smile:. I tested and the serialization is done without the `q` value so when we load the serialized value we receive everything but the `q`. When we try to load a serialized X9.42 we fail due to unknown error.